doc/tutorial/intro-to-autoinstall.rst

```diff
@@ -0,0 +1,243 @@
+.. _tutorial_intro-to-autoinstall:
+
+Introduction to Autoinstall
+***************************
+
+Ubuntu installation automation is performed with the autoinstall format.
+You might also know this feature as unattended or handsoff or preseeded
+installation.
+
+This format is supported in the following installers:
+ * Ubuntu Server, version 20.04 and later
+ * Ubuntu Desktop, version 23.04 and later
+
+Autoinstallation lets you answer all those configuration questions ahead of
+time with an *autoinstall config* and lets the installation process run without
+any interaction.
+
+
+Differences from debian-installer preseeding
+============================================
+
+*preseeds* are the way to automate an installer based on debian-installer
+(also known as d-i).
+
+Autoinstalls differ from preseeds in the following ways:
+ * the format is different (yaml vs debconf-set-selections)
+ * when the answer to a question is not present in a preseed, d-i stops and
+   asks the user for input. Autoinstalls are not like this:  by default, if
+   there is any autoinstall configuration at all, the installer takes the
+   default for any unanswered question (and fails if there is no default).
+ * You can designate particular sections in the configuration as "interactive",
+   which means the installer will still stop and ask about those.
+
+
+Providing the autoinstall configuration
+=======================================
+
+There are 2 methods of providing the autoinstall configuration:
+ * Carried as part of cloud-config
+ * Directly on the install media
+
+Autoinstall by way of cloud-config
+----------------------------------
+
+The suggested method of providing autoinstall to the Ubuntu installer is by way
+of cloud-init.  This allows the configuration to be applied to the installer
```

Comment:
You've switched from talking about cloud-config to cloud-init in a way that suggests the two are interchangeable (this might be confusing)

Reply:
Both the Subiquity and Cloud-init teams are interested in reducing confusion here.  Phrasing suggestions very welcome.  I suppose we need to resolve the title of this section first though.

Reply:
I might wait to see what Chad throws into the mix from the other direction, and have a few possible suggestions for you on Monday :)

Reply:
Chad did have a suggestion, which is now incorporated in this PR.